config/locales/en.yml

```diff
@@ -1888,6 +1888,12 @@ en:
       unacceptable_client_explanation_under_18_html:
         - For controlled work, the finances of applicants under 18 are only assessed if they are maintained or cared for by somebody who can also be included in the means assessment. Page 32 of the <a href='https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1157029/Lord_Chancellor_s_guide_to_determining_financial_eligibility_for_controlled_work_and_family_mediation_May_2023.pdf#page=32' target='_blank'>Lord Chancellor’s guidance (opens in a new tab)</a> has more details about how to assess a child’s means.
         - For certificated work, see page 94 of the <a href='https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1157030/Lord_Chancellor_s_guide_to_determining_financial_eligibility_for_certificated_work_May_2023__002_.pdf#page=94' target='_blank'>Lord Chancellor’s guidance (opens in a new tab)</a> for more details about when and how to assess a child’s means.
+      client_under_18_para_1: "Clients aged under 18 do not need a means assessment for controlled work matters unless one of the following applies:"
+      client_under_18_bullets:
+        - their means are aggregated with another person’s, like a parent or guardian
```

Comment:
I know this is directly from the the ticket but I don't think `person’s` requires an apostrophe

Reply:
In this case is does - it's a "possessive" apostrophe, used to indicated that the means belongs to a person. Without an apostrophe it would simply be a plural, meaning more than one person.

Reply:
oh yeah, it just scanned weirdly in my head